demos/link-prediction/random-walks/ctdne-link-prediction.ipynb

```diff
@@ -0,0 +1,680 @@
+      "text/plain": [
+       "<Figure size 504x504 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "temporal_node_embeddings = temporal_model.wv.vectors\n",
+    "static_node_embeddings = static_model.wv.vectors\n",
+    "plot_tsne(\"TSNE visualisation of temporal node embeddings\", temporal_node_embeddings)\n",
```

Comment:
There's no labels we can use for these to make the plots coloured, and thus more interpretable (rather than just "oh, yeah, seem grouped")?

Reply:
Yeah I guess I could omit this part, I found it slightly useful to see whether the embeddings "looked informative" on the surface while I was working on it

Reply:
I think it's reasonable to include, since as you say, it gives a hint about whether the embeddings are picking up signal in some form, I'm just wondering if there's a way we can augment them further... it seems not easily, from what's in the dataset, though.

Reply:
Theoretically, the temporal embeddings might cluster together nodes that have edges with similar time values, and separate those with very different time values. I can try to assign colours to nodes based on something like the average time across all its incident edges, but no guarantee that it will actually look meaningful

Reply:
Ah, yeah. Interesting idea, although maybe too much code to be worth it, unless you think it is going to be enough of an improvement?

Reply:
The code wasn't too much to add in the end, but I'm not sure if the visual is all that convincing:
![image](https://user-images.githubusercontent.com/33508488/76476703-02323a80-6457-11ea-8b6d-b794ef527227.png)